sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/CallAutomationAsyncClient.java

```diff
@@ -296,7 +296,8 @@ private MediaStreamingConfigurationInternal getMediaStreamingConfigurationIntern
                     mediaStreamingOptions.getContentType().toString()))
             .setTransportType(
                 MediaStreamingTransportTypeInternal.fromString(
-                    mediaStreamingOptions.getTransportType().toString()));
+                    mediaStreamingOptions.getTransportType().toString()))
+            .setStartMediaStreaming(mediaStreamingOptions.isStartMediaStreamingEnabled());
```

Comment:
@fariba625 we  have added the media streaming here with the existing alpha3 apispec. If i remember correctly you mentioned we will not be providing option to user to set startmediastreaming option, can you please let us know whether we needs to add this here or not?